client.go

```diff
@@ -192,13 +192,13 @@ func NewClient(dbDir string, cfg *ClientConfig) (*Client, func(), error) {
 }
 
 // FetchSwaps returns all loop in and out swaps currently in the database.
-func (s *Client) FetchSwaps() ([]*SwapInfo, error) {
-	loopOutSwaps, err := s.Store.FetchLoopOutSwaps()
+func (s *Client) FetchSwaps(ctx context.Context) ([]*SwapInfo, error) {
```

Comment:
What I mean is that we could perhaps wrap the `ctx` to a timeout context (per call to the store), to avoid hanging if the DB is for any reason unresponsive.